controller_stopper/package.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>controller_stopper</name>
+  <version>0.0.0</version>
+  <description>A small helper node that stops and restarts ROS controllers based on a boolean status topic.</description>
+
+  <author>Felix Exner</author>
+
+  <maintainer email="grosse@fzi.de">Marvin Große Besselmann</maintainer>
+  <maintainer email="lovro.ivanov@gmail.com">Lovro Ivanov</maintainer>
+  <maintainer email="denis.stogl@picknik.ai">Denis Stogl</maintainer>
+  <maintainer email="zelenak@picknik.ai">Andy Zelenak</maintainer>
+
+  <license>Apache License 2.0</license>
```

Comment:
We should make sure this is a valid SPDX identifier. However, let's first wait for #277 before changing that.